ODIN_II/SRC/netlist_utils.cpp

```diff
@@ -277,7 +277,8 @@ nnet_t* allocate_nnet() {
     nnet_t* new_net = (nnet_t*)my_malloc_struct(sizeof(nnet_t));
 
     new_net->name = NULL;
-    new_net->driver_pin = NULL;
+    new_net->driver_pins = NULL;
```

Comment:
make sure to adjust the free_nnet function

Reply:
Good catch